src/config/Converters.cpp

```diff
@@ -0,0 +1,125 @@
+#include "Converters.hpp"
+#include <cctype>
+#include <cstddef>
+#include <limits>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+
+std::size_t convert::toSizeT(const std::string& str)
+{
+  std::stringstream sstream(str);
+  std::size_t value = 0;
+  sstream >> value;
+
+  if (sstream.fail() || !sstream.eof()) {
+    throw std::invalid_argument("invalid number: " + str);
+  }
+
+  return value;
+}
+
+long convert::toLong(const std::string& str)
+{
+  std::stringstream sstream(str);
+  long value = 0;
+  sstream >> value;
+
+  if (sstream.fail() || !sstream.eof()) {
+    throw std::invalid_argument("invalid long: " + str);
+  }
+
+  return value;
+}
+
+std::size_t convert::toMaxBodySize(const std::string& str)
+{
+  std::size_t const len = str.length();
+  char const suffix = str[len - 1];
```

Comment:
style: We always use west const.
```suggestion
  const std::size_t len = str.length();
  const char suffix = str[len - 1];
```